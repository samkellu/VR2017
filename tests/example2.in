./binary_files/02.bin 0xFF 0xAA 0x11
