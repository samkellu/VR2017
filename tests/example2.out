Delimiter byte 0 is: 255
Delimiter byte 1 is: 170
Delimiter byte 2 is: 17
Checksum is: 68

Chunk: 0 at offset: 0
    Packet: 0
        Data before swizzle -> B0: 123, B1: 222, B2: 113
        Swizzle: XZY
        Data after swizzle -> X: 123, Y: 113, Z: 222
    Packet: 1
        Data before swizzle -> B0: 34, B1: 245, B2: 62
        Swizzle: ZYX
        Data after swizzle -> X: 62, Y: 245, Z: 34
        Ignoring packet. X: 62. Previous valid packet's X: 123. 61 > 25.
    Packet: 2
        Data before swizzle -> B0: 119, B1: 200, B2: 147
        Swizzle: YZX
        Data after swizzle -> X: 147, Y: 119, Z: 200
    Chunk Average X: 135.00, Average Y: 116.00, Average Z: 211.00

Chunk: 1 at offset: 19
    Packet: 0
        Data before swizzle -> B0: 99, B1: 88, B2: 77
        Swizzle: XYZ
        Data after swizzle -> X: 99, Y: 88, Z: 77
    Chunk Average X: 99.00, Average Y: 88.00, Average Z: 77.00

